### ISSUE TITLE
8K video in the list, bitrate, codec and fps

### ISSUE THREAD

**Issue by regs01** [feller-prj/extractor-project#18]
8K video appears in the end of the list below of worst quality and marked as 2160p.

And can you also make option to show codec, bitrate and fps. Youtube-dl does provide that info. There could be different codec and fps for same container.

**Comment by emericg** [comment]
Yes indeed, here is one 8K video for instance https://www.youtube.com/watch?v=lvf7XwUTT0I
